src/autolabel/tasks/base.py

```diff
@@ -51,6 +51,7 @@ class BaseTask(ABC):
     def __init__(self, config: AutolabelConfig) -> None:
         self.config = config
         self.image_col = self.config.image_column()
+        self.image_cols = self.config.image_columns()
```

Comment:
Would be best to merge these two together and deduplicate their references in the PR.